package.json

```diff
@@ -1,32 +1,37 @@
 {
   "name": "utsa-college",
-  "version": "0.1.0",
+  "version": "0.1.1",
   "homepage": "https://github.com/utsa-asc/college-dls",
   "license": "ISC",
-  "dependencies": {
-    "@frctl/fractal": "^1.5.11",
-    "@frctl/mandelbrot": "^1.9.4",
-    "bootstrap": "5.0.2",
-    "gulp-notify": "^4.0.0"
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/utsa-asc/college-dls.git"
   },
   "scripts": {
-    "start": "gulp dev",
-    "build": "gulp",
+    "start": "gulp",
+    "styles": "gulp styles",
```

Comment:
makes sense, keeping the build as simple as possible (without adding it into gulp)++, and making the dev serve the default tasks makes more sense, I like!

Reply:
Yeah, the more I thought about it, the more I realized what role Gulp played in our overall build process.
I decided to keep it super simple and we can add tasks as needed as we get closer to the final build.